### PR TITLE
fix(operator): migrations

### PR DESCRIPTION
**Pull request by eliobischof**

### CODE REVIEW

File: migrations/cockroach-secure/V1.1__eventstore.sql [hunk 1]
```diff
@@ -27,3 +28,4 @@ CREATE TABLE eventstore.events (
     INDEX agg_type_agg_id (aggregate_type, aggregate_id),
     CONSTRAINT previous_sequence_unique UNIQUE (previous_sequence DESC)
 );
+ALTER SEQUENCE eventstore.event_seq OWNED BY eventstore.events.event_sequence;
```
Comment (eliobischof) [inline migrations/cockroach-secure/V1.1__eventstore.sql:31]: @adlerhurst have you added this line when it was in the orbos repo?

**Comment by fforootd** [comment]
@eliobischof @stebenz FYI in the ZITADEL PR we need to name the PR Title according to **semrel**

I did change the title

File: migrations/cockroach-secure/V1.1__eventstore.sql (hunk 1 above)
Comment (adlerhurst) [reply migrations/cockroach-secure/V1.1__eventstore.sql:31]: nope ive never used `owned by`

**Review by adlerhurst — COMMENTED** [review]
In my opinion we don't need a seperate folder for secure cockroach.
if you create a user in insecure with de following line it should work too:
`CREATE USER <USER_NAME> WITH PASSWORD NULL;`

**Pull request merged**